vmaas_sync/advisory_sync.go

```diff
@@ -286,6 +287,8 @@ func checkAdvisoriesCount() error {
 
 	errataCount := int64(errataResponse.GetPages()) + 1
 	if databaseAdvisoriesCount != errataCount {
+		mismatch := errataCount - databaseAdvisoriesCount
+		advisoriesCountMismatch.Add(math.Abs(float64(mismatch)))
```

Comment:
This practically shouldn't happen. I would add just error log here:
~~~go
utils.Log("mismatch", mismatch).Error("Incremental advisories sync mismatch found!").
~~~
What do you mean?

Reply:
Jiří thought it would be nice to have and see it in Grafana, we can discuss this.

Reply:
@AlesKas ok, so can you at least add this log line, please?